yt_dlp/extractor/picarto.py

```diff
@@ -10,7 +10,8 @@
 
 
 class PicartoIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[a-zA-Z0-9]+)'
+    IE_NAME = 'picarto:base'
+    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[^/#?]+)$'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[^/#?]+)/?(?:$|[?#])'
```
any specific reason for changing `[a-zA-Z0-9]+` to `[^/#?]+` ? if you have evidence that the former is insufficent please add an `only_matching` test

Reply:
There is no specific reason. I wanted to make it flexible. I've updated it now as you suggested. Thanks.